ShipEngineSDK.Test/Api/AccountApiTests.cs

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
We should not commit auto-generated tests - especially ones that don't do anything.